lib/lure/git.go

```diff
@@ -36,14 +37,24 @@ func GitClone(auth Authentication, source string, to string) (GitRepo, error) {
 		return repo, err
 	}
 
+	var workingPath strings.Builder
+	workingPath.WriteString(to)
```

Comment:
is `to` guaranteed to end with a `/`? If not I'd add a check

Reply:
it's guaranteed _not_ to end with a `/` because of this line https://github.com/coveo/lure/pull/35/files#diff-0760eb2f6438cb24bb0f7e479da79d1aR56